.env

```diff
@@ -27,8 +27,8 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7&charset=utf8mb4"
-DATABASE_URL="postgresql://symfony:ChangeMe@127.0.0.1:5432/app?serverVersion=13&charset=utf8"
+ DATABASE_URL="mysql://freedb_yellow_green:!RDZGb3@zyySXyz@sql.freedb.tech:3306/freedb_bonheur_et_paillette?serverVersion=5.7&charset=utf8mb4"
```

Comment:
Password in clear inside the repository, please fix it asap

Reply:
Sorry, i resolved it.